src/containers/SimpleRestContainer.js

```diff
@@ -72,7 +72,21 @@ module.exports = class SimpleRestContainer extends BaseContainer {
 
         (pingComplete) => {
           if (this.caps[Capabilities.SIMPLEREST_PING_URL]) {
-            this._waitForPingUrl(this.caps[Capabilities.SIMPLEREST_PING_URL]).then(() => pingComplete()).catch(pingComplete)
+            const uri = this.caps[Capabilities.SIMPLEREST_PING_URL]
+            const verb = this.caps[Capabilities.SIMPLEREST_PING_VERB]
+            const timeout = this.caps[Capabilities.SIMPLEREST_PING_TIMEOUT]
+            try {
+              let body = JSON.stringify(this.caps[Capabilities.SIMPLEREST_PING_BODY] || {})
```

Comment:
Das setzt voraus, dass sich in dieser capability JSON befindet, kann aber auch ein String sein - bitte eine Überprüfung einbauen. Und statt "let" bitte "const" verwenden, wie sonst auch überall ...